### PR TITLE
Translate post Announcing 2.0 #133

### PR DESCRIPTION
**Pull request by yogs22**
Saya mencoba translate Post Announcing 2.0, semoga bermanfaat & Happy Hacking in October :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "vuejs.org",
   "private": true,
   "hexo": {
-    "version": "3.9.0"
+    "version": "3.8.0"
```
Comment (mazipan) [inline package.json]: Ini kenapa di downgrade

**Review by mazipan — CHANGES_REQUESTED** [review]
Fix packge.json first ya

**Comment by yogs22** [comment]
Maaf lupa nggak saya ignore

**Comment by yogs22** [comment]
sudah saya upgrade lagi versi hexonya

**Review by mazipan — APPROVED** [review]

**Pull request merged**